server.go

```diff
@@ -149,6 +149,7 @@ func populateServerConfig(config *Config) *Config {
 		HandshakeTimeout:                      handshakeTimeout,
 		IdleTimeout:                           idleTimeout,
 		AcceptSTK:                             vsa,
+		KeepAlive:                             config.KeepAlive,
```

Comment:
Can you please add a test for this value in the two tests here: https://github.com/lucas-clemente/quic-go/blob/master/server_test.go#L344-L373.